---
float32 avg_linear_x
float32 avg_angular_z
